elastic/tests/runners/snapshot_test.py

```diff
@@ -0,0 +1,198 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from unittest import mock
+
+import pytest
+from shared.runners import snapshot
+from tests import as_future
+
+
+@mock.patch("elasticsearch.Elasticsearch")
+@pytest.mark.asyncio
+async def test_mount_snapshot(es):
+    es.snapshot.get.return_value = as_future(
+        {
+            "snapshots": [
+                {
+                    "snapshot": "eventdata-snapshot",
```

Comment:
this workload should be put to rest, maybe we rename to something else?

Reply:
Addressed in https://github.com/elastic/rally-tracks/pull/302/commits/b7600440fcd38ad5fb5c001ce1e861eb17bf30b0